docker-compose fails w/ Maven dependency error

# Problem

When I run `docker-compose up` I get a Maven dependency error.

# Environment

<details><summary>Git details</summary>
<p>

```
$ git branch --show-current
develop

$ git describe
0.0.7-14-g6c17b27

$ git rev-parse HEAD
6c17b2711ab49edd6baf14ce20b5e831d6c6a419
```

</p>
</details>

<details><summary>uname -a</summary>
<p>

This is on an M1 Mac.

```
$ uname -a
Darwin H42YX2V7J0 21.6.0 Darwin Kernel Version 21.6.0: Wed Aug 10 14:28:23 PDT 2022; root:xnu-8020.141.5~2/RELEASE_ARM64_T6000 arm64
```

</p>
</details>

<details><summary>docker-compose --version</summary>
<p>

```
$ docker-compose --version
docker-compose version 1.29.2, build 5becea4c
```

</p>
</details>

# Logs

The specific error:

```
#9 [base 5/6] RUN ./mvnw dependency:resolve
#9 sha256:6c7f295343e1fa0d0ad92646b963f68a003363f049bae80174e5628c32599d6a
#9 0.400 Error: Could not find or load main class org.apache.maven.wrapper.MavenWrapperMain
#9 0.400 Caused by: java.lang.ClassNotFoundException: org.apache.maven.wrapper.MavenWrapperMain
#9 ERROR: executor failed running [/bin/sh -c ./mvnw dependency:resolve]: exit code: 1
```

<details><summary>Full Log Output</summary>
<p>

Here is the full log output from running `docker-compose up`.

```
$ docker-compose --log-level DEBUG --ansi never up 2>&1 | tee docker-compose_up.log
Using configuration files: ./docker-compose.yaml
Building service
#1 [internal] load build definition from dev.dockerfile
#1 sha256:e173fb1d530668fed4977b868c9d06e35a79768c0033808c7fb21bea3e659066
#1 transferring dockerfile: 41B done
#1 DONE 0.0s

#2 [internal] load .dockerignore
#2 sha256:0179b892986d1ca3fa4927327ed5dba55152b5ecc8042011a90d0611ae548d4f
#2 transferring context: 2B done
#2 DONE 0.0s

#3 [internal] load metadata for docker.io/azul/zulu-openjdk:11
#3 sha256:ec6ce0188337f769c9880d02d4163b649bfdbf2e70b8504f6ee891e296aa275a
#3 DONE 0.6s

#4 [base 1/6] FROM docker.io/azul/zulu-openjdk:11@sha256:6a74dc6078d79fbf4eafa9fc23b176ed791e3fda79b510288dfa8f522d2836fc
#4 sha256:8850bc2f2a552e02657d57d166744362dbdcb52ba493fcbc60d0358784404f87
#4 DONE 0.0s

#6 [internal] load build context
#6 sha256:49d72ee4f4b3999d9ab865d6a87c90563288714d421ad9551afdb92fdfe223a4
#6 transferring context: 5.03kB 0.0s done
#6 DONE 0.0s

#5 [base 2/6] WORKDIR /app
#5 sha256:51f4ca31f49c406a5cffcf71e52d6b8d55a4683646bb050fc7bf2a7e21628c0e
#5 CACHED

#7 [base 3/6] COPY .mvn/ .mvn
#7 sha256:4cd21ef7afccb7f21afe8beb62650519e4f56c6017afa3e670ba85c6c19e9da5
#7 CACHED

#8 [base 4/6] COPY mvnw pom.xml ./
#8 sha256:b32fdbca0370d7acaeeba86b80db712f864723368cd9adfbe814487112989884
#8 CACHED

#9 [base 5/6] RUN ./mvnw dependency:resolve
#9 sha256:6c7f295343e1fa0d0ad92646b963f68a003363f049bae80174e5628c32599d6a
#9 0.400 Error: Could not find or load main class org.apache.maven.wrapper.MavenWrapperMain
#9 0.400 Caused by: java.lang.ClassNotFoundException: org.apache.maven.wrapper.MavenWrapperMain
#9 ERROR: executor failed running [/bin/sh -c ./mvnw dependency:resolve]: exit code: 1
------
 > [base 5/6] RUN ./mvnw dependency:resolve:
------
executor failed running [/bin/sh -c ./mvnw dependency:resolve]: exit code: 1
Service 'service' failed to build : Build failed
```

</p>
</details>

--- a/dev.dockerfile
+++ b/dev.dockerfile
@@ -1,4 +1,8 @@
 FROM azul/zulu-openjdk:11 as base
+
+RUN apt update -y
+RUN apt install wget -y
+
 WORKDIR /app
 COPY .mvn/ .mvn
 COPY mvnw pom.xml ./

--- a/src/main/resources/application-dev.properties
+++ b/src/main/resources/application-dev.properties
@@ -17,7 +17,7 @@ spring.datasource.password=${POSTGRES_PASSWORD}
 spring.datasource.url=jdbc:postgresql://db/l0_auth
 
 # Sendgrid
-com.mytiki.l0_auth.sendgrid.apikey=${SENDGRID_API_KEY}
+com.mytiki.l0_auth.sendgrid.apikey=${SENDGRID_API_KEY:''}
 
 # JWT
 com.mytiki.l0_auth.jwt.private_key=MEECAQAwEwYHKoZIzj0CAQYIKoZIzj0DAQcEJzAlAgEBBCBOfRkEaPbm/5RkT5rxHblmJay/fzqsOgbunyE/dhIWkg==

